azure-pipelines.yml

```
@@ -4,7 +4,7 @@ pr: none
 trigger: none
 
 variables:
-  - group: 'Github and NPM secrets'
+  - group: 'GitHub and NPM secrets'
```

Please revert this change - this could break the release pipeline

---

What do you think about using a suggestion in the future?

```suggestion
  - group: 'Github and NPM secrets'
```